contact.md

```diff
@@ -23,38 +24,18 @@ Some important channels:
 Our mailing list for developers working on Tekton is
 [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev). All docs
 which are shared with the Tekton community (including
-[working group](#working-group) meeting recordings!) are made visible to this
+[working groups](./working-groups.md) meeting recordings!) are made visible to this
 group, so please join if you are interested in accessing those docs.
 
 Our mailing list for end users of Tekton is
 [tekton-users@](https://groups.google.com/forum/#!forum/tekton-users).
 
 ## Calendar
 
-Tekton events such as [working group](#working-group) and [other meetings](#other-meetings)
+Tekton events such as [working group](./working-groups.md) and [other meetings](#other-meetings)
```

Comment:
a bit of scope creep but it looks like `#other-meetings` doesnt exist anymore

Reply:
oups :sweat_smile: